.github/workflows/cd-docs.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
       - name: Install tox
         run: |
-          pip install --upgrade pip==24.0 uv==0.2.13 tox tox-uv==1.9.0
+          pip install --upgrade pip==24.1 uv==0.2.17 tox tox-uv==1.9.0
```

Comment:
pip install --upgrade pip==24.1.1 uv==0.2.17
uv --version
uv pip install tox tox-uv==1.9.0

Reply:
might as well use uv since it's installed

Reply:
We use uv solely for making our tox commands faster & not for installing global packages. Hence we want to keep uv & tox global.